docs/settings.md

```diff
@@ -60,6 +60,31 @@ output_matplotlib_strings
     - `"remove-warn"` (default) or `"remove-error"`: remove all matplotlib strings in outputs, and log a warning or error
     - `"warn"` or "error": log a warning or error if matplotlib strings in outputs
 
+## Markdown Parsing Settings
+
+Adding an object of `parser` to the settings will allow you to control various parser behaviors.
+
+```{code-block} yaml
+:filename: myst.yml
+project:
+  settings:
+    parser:
+      dollarmath: false
+```
+
+:::{warning}
+Markdown parsing settings currently only work on the `myst.yml` project settings, not the page frontmatter.
```

Comment:
Right now this is only in the project, in future we could have there settings in the frontmatter of the files, but that would require pre-parsing that part, and we aren't setup for that.